src/test/java/org/prebid/server/bidder/amx/AmxBidderTest.java

```diff
@@ -215,6 +215,30 @@ public void makeBidsShouldReturnErrorIfAdmNotContainSearchPoint() throws JsonPro
                         BidderError.badServerResponse("Adm should contain vast search point in bidder: bidId"));
     }
 
+    @Test
+    public void makeBidsShouldSkipBidAndAddErrorIfFailedToParseBidExt() throws JsonProcessingException {
+        // given
+        final ObjectNode bidExt = mapper.createObjectNode();
+        bidExt.put("startdelay", "2");
+        final HttpCall<BidRequest> httpCall = givenHttpCall(BidRequest.builder().build(),
+                mapper.writeValueAsString(
+                        givenBidResponse(bidBuilder -> bidBuilder
+                                .id("bidId")
+                                .adm("</Impression>")
+                                .ext(mapper.createObjectNode().set("startdelay", mapper.createObjectNode())))));
+
+        // when
+        final Result<List<BidderBid>> result = amxBidder.makeBids(httpCall, null);
+
+        // then
+        assertThat(result.getErrors())
+                .containsExactly(
+                        BidderError.badInput("Cannot deserialize instance of `java.lang.String` out "
+                                + "of START_OBJECT token\n at [Source: UNKNOWN; line: -1, column: -1] (through "
+                                + "reference chain: org.prebid.server.bidder.amx.model.AmxBidExt[\"startdelay\"])"));
```

Comment:
Usually in tests, we check only for start of the message
assertThat(result.getErrors()).allMatch(error -> error.getType() == BidderError.Type.bad_input
                && error.getMessage().startsWith("Cannot deserialize instance"));